--- libxcrypt-4.4.17/Makefile.am.orig	2020-08-23 09:49:51.000000000 +0200
+++ libxcrypt-4.4.17/Makefile.am	2020-08-23 10:47:59.294128962 +0200
@@ -84,19 +84,15 @@
 	test/des-cases.h \
 	test/ka-table.inc
 
-if ENABLE_XCRYPT_COMPAT_FILES
 nodist_include_HEADERS += xcrypt.h
-else
-nodist_noinst_HEADERS += xcrypt.h
-endif
 
 noinst_PROGRAMS = \
 	lib/gen-des-tables
 
 lib_LTLIBRARIES = \
-	libcrypt.la
+	libxcrypt.la
 
-libcrypt_la_SOURCES = \
+libxcrypt_la_SOURCES = \
 	lib/alg-des-tables.c \
 	lib/alg-des.c \
 	lib/alg-gost3411-2012-core.c \
@@ -143,23 +139,23 @@
 
 # Build libcrypt.so.2 if obsolete APIs are excluded, libcrypt.so.1 otherwise.
 if ENABLE_OBSOLETE_API
-libcrypt_la_VERSION = 2:0:1
+libxcrypt_la_VERSION = 3:0:1
 else
-libcrypt_la_VERSION = 2:0:0
+libxcrypt_la_VERSION = 3:0:0
 endif
 
-libcrypt_la_LDFLAGS = -version-info $(libcrypt_la_VERSION)
+libxcrypt_la_LDFLAGS = -version-info $(libxcrypt_la_VERSION)
 
 if HAVE_VSCRIPT
-libcrypt_la_LDFLAGS += $(VSCRIPT_LDFLAGS),$(builddir)/libcrypt.map
+libxcrypt_la_LDFLAGS += $(VSCRIPT_LDFLAGS),$(builddir)/libcrypt.map
 endif
 
-libcrypt_la_LDFLAGS += $(UNDEF_FLAG) $(TEXT_RELOC_FLAG) $(AM_LDFLAGS)
+libxcrypt_la_LDFLAGS += $(UNDEF_FLAG) $(TEXT_RELOC_FLAG) $(AM_LDFLAGS)
 
-libcrypt_la_CPPFLAGS = $(AM_CPPFLAGS) -DIN_LIBCRYPT
+libxcrypt_la_CPPFLAGS = $(AM_CPPFLAGS) -DIN_LIBCRYPT
 
 CONFIG_STATUS_DEPENDENCIES = lib/libcrypt.minver.$(MINVER_OS)
-EXTRA_libcrypt_la_DEPENDENCIES = libcrypt.map
+EXTRA_libxcrypt_la_DEPENDENCIES = libcrypt.map
 CLEANFILES = \
 	Makefile.deps Makefile.deps.T \
 	libcrypt.map libcrypt.map.T \
@@ -234,9 +230,9 @@
 	uninstall-hook-xcrypt-static
 install-exec-hook-xcrypt-static:
 	cd $(DESTDIR)$(libdir) && \
-		$(LN_S) libcrypt.a libxcrypt.a
+		$(LN_S) libxcrypt.a libcrypt.a
 uninstall-hook-xcrypt-static:
-	-rm -f $(DESTDIR)$(libdir)/libxcrypt.a
+	-rm -f $(DESTDIR)$(libdir)/libcrypt.a
 endif
 
 if ENABLE_SHARED
@@ -249,9 +245,9 @@
 	uninstall-hook-xcrypt-shared
 install-exec-hook-xcrypt-shared:
 	cd $(DESTDIR)$(libdir) && \
-		$(LN_S) libcrypt.so libxcrypt.so
+		$(LN_S) libxcrypt.so libcrypt.so
 uninstall-hook-xcrypt-shared:
-	-rm -f $(DESTDIR)$(libdir)/libxcrypt.so
+	-rm -f $(DESTDIR)$(libdir)/libcrypt.so
 endif
 endif
 
@@ -278,7 +274,7 @@
 	uninstall-hook-libstatic
 install-exec-hook-libstatic:
 	cd $(DESTDIR)$(libdir) && \
-		$(LN_S) libcrypt.a libowcrypt.a
+		$(LN_S) libxcrypt.a libowcrypt.a
 uninstall-hook-libstatic:
 	-rm -f $(DESTDIR)$(libdir)/libowcrypt.a
 endif
@@ -291,8 +287,8 @@
 	uninstall-hook-libshared
 install-exec-hook-libshared:
 	cd $(DESTDIR)$(libdir) && \
-		$(LN_S) libcrypt.so libowcrypt.so && \
-		$(LN_S) libcrypt.so.1 libowcrypt.so.1
+		$(LN_S) libxcrypt.so libowcrypt.so && \
+		$(LN_S) libxcrypt.so.1 libowcrypt.so.1
 uninstall-hook-libshared:
 	-rm -f  $(DESTDIR)$(libdir)/libowcrypt.so \
 		$(DESTDIR)$(libdir)/libowcrypt.so.1
@@ -401,7 +397,7 @@
 endif
 
 if ENABLE_OBSOLETE_API
-libcrypt_la_SOURCES += lib/crypt-des-obsolete.c
+libxcrypt_la_SOURCES += lib/crypt-des-obsolete.c
 check_PROGRAMS += test/des-obsolete test/des-obsolete_r test/fcrypt-enosys
 endif
 
@@ -421,7 +417,7 @@
 
 AM_TESTS_ENVIRONMENT = \
     host_os="$(host_os)"; \
-    lib_la="./libcrypt.la"; \
+    lib_la="./libxcrypt.la"; \
     lib_map="$(srcdir)/lib/libcrypt.map.in"; \
     AWK="$(AWK)"; \
     CC="$(CC)"; \
@@ -438,7 +434,7 @@
 # Refer to object files that are linked into libxcrypt with the
 # qualified name, lib/libcrypt_la-foobar.lo, to prevent them from
 # being compiled a second time.
-COMMON_TEST_OBJECTS = lib/libcrypt_la-crypt-common.lo libcrypt.la
+COMMON_TEST_OBJECTS = lib/libxcrypt_la-crypt-common.lo libxcrypt.la
 
 test_badsalt_LDADD = $(COMMON_TEST_OBJECTS)
 test_badsetting_LDADD = $(COMMON_TEST_OBJECTS)
@@ -474,58 +470,58 @@
 # These tests call internal APIs that may not be accessible from the
 # fully linked shared library.
 test_alg_des_LDADD = \
-	lib/libcrypt_la-alg-des.lo \
-	lib/libcrypt_la-alg-des-tables.lo \
+	lib/libxcrypt_la-alg-des.lo \
+	lib/libxcrypt_la-alg-des-tables.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_gost3411_2012_LDADD = \
-	lib/libcrypt_la-alg-gost3411-2012-core.lo \
-	lib/libcrypt_la-alg-gost3411-2012-hmac.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-core.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-hmac.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_gost3411_2012_hmac_LDADD = \
-	lib/libcrypt_la-alg-gost3411-2012-core.lo \
-	lib/libcrypt_la-alg-gost3411-2012-hmac.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-core.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-hmac.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_hmac_sha1_LDADD = \
-	lib/libcrypt_la-alg-sha1.lo \
-	lib/libcrypt_la-alg-hmac-sha1.lo \
+	lib/libxcrypt_la-alg-sha1.lo \
+	lib/libxcrypt_la-alg-hmac-sha1.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_md4_LDADD = \
-	lib/libcrypt_la-alg-md4.lo \
+	lib/libxcrypt_la-alg-md4.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_md5_LDADD = \
-	lib/libcrypt_la-alg-md5.lo \
+	lib/libxcrypt_la-alg-md5.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_pbkdf_hmac_sha256_LDADD = \
-	lib/libcrypt_la-alg-sha256.lo \
+	lib/libxcrypt_la-alg-sha256.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_sha1_LDADD = \
-	lib/libcrypt_la-alg-sha1.lo \
+	lib/libxcrypt_la-alg-sha1.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_sha256_LDADD = \
-	lib/libcrypt_la-alg-sha256.lo \
+	lib/libxcrypt_la-alg-sha256.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_sha512_LDADD = \
-	lib/libcrypt_la-alg-sha512.lo \
+	lib/libxcrypt_la-alg-sha512.lo \
 	$(COMMON_TEST_OBJECTS)
 test_alg_yescrypt_LDADD = \
-	lib/libcrypt_la-alg-sha256.lo \
-	lib/libcrypt_la-alg-yescrypt-common.lo \
-	lib/libcrypt_la-alg-yescrypt-opt.lo \
+	lib/libxcrypt_la-alg-sha256.lo \
+	lib/libxcrypt_la-alg-yescrypt-common.lo \
+	lib/libxcrypt_la-alg-yescrypt-opt.lo \
 	$(COMMON_TEST_OBJECTS)
 test_crypt_gost_yescrypt_LDADD = \
-	lib/libcrypt_la-alg-gost3411-2012-core.lo \
-	lib/libcrypt_la-alg-gost3411-2012-hmac.lo \
-	lib/libcrypt_la-alg-sha256.lo \
-	lib/libcrypt_la-alg-yescrypt-common.lo \
-	lib/libcrypt_la-alg-yescrypt-opt.lo \
-	lib/libcrypt_la-crypt-yescrypt.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-core.lo \
+	lib/libxcrypt_la-alg-gost3411-2012-hmac.lo \
+	lib/libxcrypt_la-alg-sha256.lo \
+	lib/libxcrypt_la-alg-yescrypt-common.lo \
+	lib/libxcrypt_la-alg-yescrypt-opt.lo \
+	lib/libxcrypt_la-crypt-yescrypt.lo \
 	$(COMMON_TEST_OBJECTS)
 
 test_getrandom_interface_LDADD = \
-	lib/libcrypt_la-randombytes.lo \
+	lib/libxcrypt_la-randombytes.lo \
 	$(COMMON_TEST_OBJECTS)
 test_getrandom_fallbacks_LDADD = \
-	lib/libcrypt_la-randombytes.lo \
+	lib/libxcrypt_la-randombytes.lo \
 	$(COMMON_TEST_OBJECTS)
 
 if HAVE_LD_WRAP
--- libxcrypt-4.4.10/lib/libxcrypt.pc.in.orig	2019-09-18 18:07:41.000000000 +0200
+++ libxcrypt-4.4.10/lib/libxcrypt.pc.in	2019-10-05 20:26:31.028284316 +0200
@@ -11,5 +11,5 @@
 Name: @PACKAGE@
 Version: @VERSION@
 Description: Extended crypt library for DES, MD5, Blowfish and others
-Libs: -L${libdir} -lcrypt
+Libs: -L${libdir} -lxcrypt
 Cflags: -I${includedir}
