--- libxcrypt-3.1.1/configure.ac.orig	2015-05-13 01:29:46.000000000 +0200
+++ libxcrypt-3.1.1/configure.ac	2017-04-10 18:59:27.731520714 +0200
@@ -38,7 +38,7 @@
     EXTRA_CFLAGS="$EXTRA_CFLAGS -Wcast-qual -Winline"
     EXTRA_CFLAGS="$EXTRA_CFLAGS -Wmissing-declarations -Wmissing-prototypes"
     EXTRA_CFLAGS="$EXTRA_CFLAGS -Wnested-externs -Wshadow -Wstrict-prototypes"
-    EXTRA_CFLAGS="$EXTRA_CFLAGS -Wundef -Wpointer-arith -Werror"
+    EXTRA_CFLAGS="$EXTRA_CFLAGS -Wundef -Wpointer-arith"
 fi
 AC_SUBST(EXTRA_CFLAGS)
 
